### ISSUE TITLE
Different icons for audio outputs

### ISSUE THREAD

**Issue by loki1368** [Jas-SinghFSU/HyprPanel#203]
**Is your feature request related to a problem? Please describe.**
I use two audio outputs alternately and would like to see which one is currently being used

**Describe the solution you'd like**
I would like to have an ability for select icon for each audio output

**Comment by Jas-SinghFSU** [comment]
Are you meaning that you want custom icons in the volume module in the bar?

**Comment by loki1368** [comment]
Yes, view custom icons in the bar. In example, speaker icon for speakers output and headphones icon for headphones.
